connector-splunk/src/main/resources/application.properties

```diff
@@ -21,12 +21,8 @@ camel.component.kafka.sasl-mechanism=GSSAPI
 camel.component.kafka.security-protocol=PLAINTEXT
 camel.component.kafka.ssl-truststore-location=
 camel.component.kafka.ssl-truststore-type=JKS
-camel.component.kafka.poll-on-error=RECONNECT
 camel.component.kafka.retries=3
 camel.component.kafka.retry-backoff-ms=200
-# https://camel.apache.org/manual/camel-3x-upgrade-guide-3_17.html#_camel_kafka
-camel.component.kafka.allow-manual-commit=true
-camel.component.kafka.kafka-manual-commit-factory=#class:org.apache.camel.component.kafka.consumer.DefaultKafkaManualCommitFactory
```

Comment:
This was copied from the Eventing app. I wasn't sure why it was introduced initially but now I'm sure I don't want it anymore.